src/components/reservation/forms/NoInterestAfterModal.tsx

```diff
@@ -33,6 +33,7 @@ const NoInterestAfterModal = ({
       items={formatInterestPeriods}
       defaultSelectedItem={String(selectedInterest)}
       selectHandler={(value: string) => setSelectedInterest(Number(value))}
+      ariaLabel={t("modalReservationFormNoInterestAfterHeaderTitleText")}
```

Comment:
I understand what you are getting at with this approach of duplicating the title but I do not think it makes for a very good solution.

Either we should:

- Create a separate text entry for the dropdown aria label `modalReservationFormNoInterestAfterLabelText`
- Mark the dropdown as labelled by the header of the modal using `aria-labelled-by`. This requires setting a unique id for the modal header and passing this to the children. It gets somewhat complex so if you want to go that route come talk to me about it.